### DISCUSSION TITLE
Problem installing via Meson and Ninja

### DISCUSSION THREAD

**Discussion by MicheleD1999** [mesonbuild/meson#12752] [Q&A]
Hello to everybody, I'm kinda new on the Mac environment so I'm sorry if my problem is easily solvable and I haven't figured it out yet. I'm currently installing the package Kira (https://gitlab.com/kira-pyred/kira.git) on my macOS M2 Sonoma 14.2.1, following the suggested meson build.

I firstly installed meson with 
pip3 install --user meson

and then I installed ninja. The problem is that when I launch in the builddir the command
ninja install
I obtained the error:

OSError: [Errno 30] Read-only file system: '/install'
ERROR: Unhandled python OSError. This is probably not a Meson bug, but an issue with your build environment.

Then I tried to uninstall with pip3 uninstall meson and also for ninja, and I installed both with homebrew:
brew install meson
brew install ninja

Currently if I launch which brew and which ninja they appear to be both in /opt/brew/bin

The problem is that again I obtain the same OSError when doing ninja install. 

Thanks to everybody,
Michele

**Comment by eli-schwartz** [comment]
```
OSError: [Errno 30] Read-only file system: '/install'
```


This happens because of the instructions at 

https://gitlab.com/kira-pyred/kira#id3

> ```
> meson setup --prefix=/install/path builddir
> ```
> 
> where builddir is the build directory. Specifying the installation prefix with --prefix is optional.

The "/install/path" is not meant to be taken literally, that README.rst is advising you using pseudo-prose to select an installation path and pass it to the --prefix argument.

It will need to be a writable location. Possibly somewhere in /opt.

**Comment by MicheleD1999** [comment]
Thank you very much, it was really a stupid error!
